### PR TITLE
fix(common): org center style optimization

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Org center style optimization.

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)
image
image
image
image

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.4

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/common/components/table/index.tsx [hunk 1]
```diff
@@ -295,10 +302,23 @@ function WrappedTable<T extends object = any>({
     type: 'page' | 'prev' | 'next' | 'jump-prev' | 'jump-next',
     originalElement: React.ReactElement<HTMLElement>,
   ) => {
+    const { total } = pagination;
     switch (type) {
       case 'prev':
+        return (
+          <div className="bg-hover" onClick={() => current > 1 && onTableChange({ pageNo: current - 1 })}>
```
Comment (daskyrk) [inline shell/app/common/components/table/index.tsx:309]: Should disable if can not click?

File: shell/app/common/components/table/index.tsx [hunk 2]
```diff
@@ -104,6 +104,7 @@ declare type TableAction = 'paginate' | 'sort' | 'filter';
 interface IPaginationProps {
   pagination: TablePaginationConfig;
   onTableChange: ([key]: any) => void;
+  hiddenPop: () => void;
```
Comment (daskyrk) [inline shell/app/common/components/table/index.tsx:107]: What does this mean ?

File: shell/app/common/components/table/index.tsx [hunk 3]
```diff
@@ -278,14 +280,19 @@ function WrappedTable<T extends object = any>({
...
       placement="top"
       overlayClassName="pagination-jump"
+      visible={goToVisible}
+      onVisibleChange={setGoToVisible}
     >
-      <div className="pagination-center bg-hover mx-1 px-3 rounded" onClick={(e) => e.stopPropagation()}>
-        {pagination?.total ? pagination?.current : 0} / {Math.ceil(pagination?.total / pagination?.pageSize)}
+      <div className="pagination-center bg-hover mx-1 px-3 rounded" onClick={() => setGoToVisible(true)}>
+        {pagination?.total ? pagination?.current : 0} /{' '}
+        {(pagination?.total && pagination?.pageSize && Math.ceil(pagination?.total / pagination?.pageSize)) || 0}
```
Comment (daskyrk) [inline shell/app/common/components/table/index.tsx:295]: Suggest add a default object if pagination is undefined.

File: shell/app/modules/dcos/pages/cluster-dashboard/resources-summary.scss [hunk 4]
```diff
@@ -7,3 +7,7 @@
     color: $color-dark-8;
   }
 }
+
+.resource-summary-desc {
+  color: $color-text-desc;
```
Comment (daskyrk) [inline shell/app/modules/dcos/pages/cluster-dashboard/resources-summary.scss:12]: Tail wind have this color ?

File: shell/app/common/components/table/index.tsx (hunk 1 above)
Comment (hujiahao-hjh) [reply shell/app/common/components/table/index.tsx:309]: Antd  will automatically add disable.

**Pull request merged**